lxd-agent/network_1.11.go

```diff
@@ -0,0 +1,13 @@
+// +build go1.11
+
+package main
+
+import (
+	"net"
+
+	"github.com/mdlayher/vsock"
```

Comment:
@monstermunchkin @stgraber is this needed anymore, did we go with "github.com/linuxkit/virtsock/pkg/vsock" in the end?

Reply:
Yes, this is needed. Exec fails even for Go 1.13 with the other vsock package.